source/develop/release-management/features/network/ipv6_subnets.html.md

```diff
@@ -0,0 +1,175 @@
+---
+title: IPv6 subnets on ovirt-provider-ovn
+category: feature
+authors: mdbarroso
+feature_name: Support IPv6 subnets on ovirt-provider-ovn
+feature_modules: ovirt-provider-ovn
+feature_status: In Development
+---
+
+# IPv6 subnets on ovirt-provider-ovn
+
+## Summary
+
+This feature provides support for IPv6 addresses on guest virtual machines
+whose networking is provided by OVN.
+
+Address auto-configuration is - despite desirable - not a must at this stage,
+and its implementation will take place at a later stage.
+
+Related patches can be found on [gerrit topic:provider_ipv6_subnets](https://gerrit.ovirt.org/#/q/topic:provider_ipv6_subnets).
+
+### Owner
+* Feature Owner: [Miguel Duarte Barroso](https://github.com/maiqueb) (mbarroso in #ovirt)
+* Email: <mdbarroso@redhat.com>
+
+### Terminology
+
+* Networking API: [OpenStack RESTful Networking API](https://developer.openstack.org/api-ref/network/v2/)
+* OVN: Open Virtual Network. OVN is an OVS (Open vSwitch) extension that
+brings Software Defined Networking to OVS.
+* oVirt OVN provider: a proxy that implements a subset of the Networking API
+and interacts with OVN.
+* subnet: a layer 3 concept, created on top of a network. As per openstack's documentation it is a 'block of IP addresses and associated configuration state. Subnets are used to allocate IP addresses when new ports are created on a network.'
+* network: an isolated Layer 2 networking segment.
+* IPAM: IP Address Management.
+
+### Benefit to oVirt
+
+IPv6 addresses are currently supported in oVirt, but the L3 capabilities on OVN
+provided networks are severelly limited, since it is missing IPAM and routing
+capabilities.
+This feature addresses those gaps, by providing IPv6 IPAM capabilities to oVirt
+logical networks, provided by OVN.
+With IPv6 IPAM in place, routing amongst different IPv6 subnets should be
+possible, and achieved through an OVN logical router.
+
+## Objectives
+
+Provide guests (virtual machines) attached to OVN networks IPv6 address support.
+The following configurations should be possible:
+
+* static addresses: the user can manually specify the ipv6 address of a port
+* stateful DHCPv6: the user can leverage OVN's native DHCPv6 service to get an ipv6 address in the guests.
+* stateless DHCPv6: the IP address allocated to the guest's interface will be created from the port's MAC address and the IP prefix being announced in the *Router Advertisement* messages. Other connection parameters - like max hops, MTU, dns configs, are set using DHCPv6.
```

Comment:
when you write: ".. will be created from..." it sounds like you are doing that and not the IPv6 stack

Reply:
Removed that from from the objectives section, redirecting the user to the IPAM section, where I first explain the 3 concepts.

Updated the text, explicitly indicating it is OVN who assigns the IP addresses.

@roniezr - is the reworked text better ?